packages/ia-components/package.json

```diff
@@ -6,7 +6,7 @@
     "scripts": {
         "storybook": "yarn run test:generate-output && yarn run storybook:start",
         "storybook:start": "start-storybook -p 9001 -c .storybook",
-        "test:generate-output": "mkdir jest-test-utils && jest --json --outputFile=jest-test-utils/jest-test-results.json || true",
+        "test:generate-output": "rm -rf jest-test-utils && mkdir jest-test-utils && jest --json --outputFile=jest-test-utils/jest-test-results.json || true",
```

Comment:
once we have a whole lot more tests, removing this may take a while. The error is a moot one as it is not a hard fail, the yarn command continues to move forward running storybook.  A suggestion would be to look for the `jest-test-utils` directory and if it isn't there, create it. This can be parsed out into its own script so its usage is composable like the others.

Reply:
Yes, @iisa I will seek a better implementation as suggested by you.